### ISSUE TITLE
New Alarm: cpu_high_watermark, Alarm Info: 100.0

### ISSUE THREAD

**Issue by aaamitsingh** [emqx/emqx-docker#126]
- [ ] BUG REPORT

-------------------------------------------------------------------------------
> BUG REPORT

![image](https://user-images.githubusercontent.com/25000347/62937983-b4cfeb00-bdeb-11e9-88d0-75db48bc72c1.png)

### EMQ version
image: emqx/emqx:latest

### Docker version

Which docker-engine version?

```bash
docker -v
Docker version 18.09.7, build 2d0083d
```

How docker info?

```bash
docker info
Containers: 1
 Running: 1
 Paused: 0
 Stopped: 0
Images: 2
Server Version: 18.09.7
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version:
runc version: N/A
init version: v0.18.0 (expected: fec3683b971d9c3ef73f284f176672c44b448662)
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.15.0-1021-aws
Operating System: Ubuntu 18.04.1 LTS
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 3.851GiB
Name: ip-172-31-28-189
ID: YKAD:JV3T:VMJB:QGZM:LO3P:DFTE:WH4X:7JVK:JSD2:M6EA:FT3R:TC3S
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

WARNING: No swap limit support
```
docker stats
![image](https://user-images.githubusercontent.com/25000347/62938154-12fcce00-bdec-11e9-8433-6caba565d28e.png)

EMQX Dashboard
![image](https://user-images.githubusercontent.com/25000347/62938231-417aa900-bdec-11e9-9f41-8ac2ddbc5c4b.png)

### System

cat /etc/os-release
NAME="Ubuntu"
VERSION="18.04.1 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.1 LTS"
VERSION_ID="18.04"

### Hardware

How is the host machine?

Instance Type - t2.medium
vCPU - 2 
Memory - 4 (GiB)

**Comment by andreacomo** [comment]
+1

**Comment by infrarosso78** [comment]
same problem here

**Comment by FabGre** [comment]
+1

**Comment by zhanghongtong** [comment]
Thanks for your feedback. This is a bug in Erlang language. We will fix this problem in the next version

**Comment by JaxSONG** [comment]
+1

**Comment by zhaochengwei** [comment]
+1

**Comment by ServerTech** [comment]
+1

**Comment by kortez06** [comment]
+1

**Comment by zhanghongtong** [comment]
We have just released version 3.2.3. We have fixed this bug. Welcome to use it

**Comment by aaamitsingh** [comment]
Thanks @zhanghongtong :)
